test/functional/apps/context/_date_nanos.js

```diff
@@ -61,19 +59,17 @@ export default function ({ getService, getPageObjects }) {
       await PageObjects.context.navigateTo(TEST_INDEX_PATTERN, TEST_ANCHOR_TYPE, 'AU_x3-TaGFA8no6Qjisd');
       await PageObjects.context.clickPredecessorLoadMoreButton();
       await PageObjects.context.clickSuccessorLoadMoreButton();
-      const table = await docTable.getTable();
-      const rows = await docTable.getBodyRows(table);
-      const actualRowsText = await Promise.all(rows.map(row => row.getVisibleText()));
+      const actualRowsText = await docTable.getRowsText();
       const expectedRowsText = [
-        'Sep 22, 2019 @ 23:50:13.253123345\n5',
-        'Sep 18, 2019 @ 06:50:13.000000104\n4',
-        'Sep 18, 2019 @ 06:50:13.000000103\n2',
-        'Sep 18, 2019 @ 06:50:13.000000102\n1',
-        'Sep 18, 2019 @ 06:50:13.000000101\n0',
-        'Sep 18, 2019 @ 06:50:13.000000001\n-1',
-        'Sep 18, 2019 @ 06:50:13.000000000\n-2',
-        'Sep 18, 2019 @ 06:50:12.999999999\n-3',
-        'Sep 19, 2015 @ 06:50:13.000100001\n1'
+        'Sep 22, 2019 @ 23:50:13.2531233455',
+        'Sep 18, 2019 @ 06:50:13.0000001044',
+        'Sep 18, 2019 @ 06:50:13.0000001032',
+        'Sep 18, 2019 @ 06:50:13.0000001021',
+        'Sep 18, 2019 @ 06:50:13.0000001010',
+        'Sep 18, 2019 @ 06:50:13.000000001-1',
+        'Sep 18, 2019 @ 06:50:13.000000000-2',
+        'Sep 18, 2019 @ 06:50:12.999999999-3',
+        'Sep 19, 2015 @ 06:50:13.0001000011'
```

Comment:
It's an interesting detail here, in seems like in the old version the column content of the row was concatenated so it was `col1\ncol2`. It's a bit odd here that this is just the case for negative values of col2 (-1, -2, -3). do you have any idea why?

Reply:
Hm, I don't see difference between concatenation of positive and negative values 😮

In the old version, we were using WebDriver API via `getVisibleText()` function on row element and in such cases (getting a text on a parent element that is actually text of its multiple children ) behaves differently in Chrome and Firefox:  chromedriver usually concatenates with `\n` and geckodriver without it - I already had to fix it in some other tests.
In this new version, we get inner html of row element and parse it with `cheerio`, which returns row text without `\n` not depending on browser:

`Sep 18, 2019 @ 06:50:12.999999999` + `-3`
`Sep 19, 2015 @ 06:50:13.000100001` + `1`

becomes
```
        'Sep 18, 2019 @ 06:50:12.999999999-3',
        'Sep 19, 2015 @ 06:50:13.0001000011'
```

As an alternative, we can parse each field text individually and compare arrays. Will it be better?

Reply:
Ah!! sorry, need more coffee this morning, it's correct, the positive number is added to the string. Didn't see the number-forest for the number-trees.  No, there's no change necessary!